### ISSUE TITLE
switching from ship to on foot and then back to ship the script fails (aborts) 

### ISSUE THREAD

**Issue by macvern** [Aussiedroid/AD-EDWarthogEnhancedScript#20]
Trying the odyssey version.  looking forward to your update.  huge fan.  thank you for all your efforts.

Best wishes from Calgary, AB

MacVern

**Comment by Aussiedroid** [comment]
Hi @macvern ,

Much appreciated mate, hope to have an update soon (tm) :)

Thanks for the report too. I will see if I can replicate this issue over the weekend and see if I can figure out why its crashing.

Could I confirm, which version of the script are you running in Odyssey? Depending on which version you are loading up, would you mind trying the other version too and see if it crashes too?

Have you made any changes yourself (I assume you have mapped some on-foot stuff, but anything else from the default scripts?)?

Is the issue replicatable? ie does it happen every time you try to go back into the ship?

Cheers, AD

**Comment by macvern** [comment]
I think I had to go back to AD 4.2 bindings but if there's a v5.0 binding that I can use with these VKB T2 Rudder pedals then I would be happy to try again.

**Comment by Aussiedroid** [comment]
I found out last night that there is new flags to be read from the status.json file from ED that were added in Odyssey. These flags track the on-foot component. What I think might be happening here is that the json reading function is failing when it tries to do the on-foot flags or reading the file. I'll keep this on my radar and look into it further at any rate.

Script should work with any rudder pedals once I track down the bug :)

AD
